andela/troupon
# Ft application responsiveness 115166123

#### What does this PR do?
This PR makes the application more responsive.

#### Description
1) The following elements/pages should render elements properly on screen widths below 480px:
* Dropdown menu for viewing cart items
* Login dropdown menu
* Font sizes

2) Elements should align properly on view more details page.

#### How should this be manually tested?
Either resize browser window to below 480 px or visit site on mobile phone and do the following:
* Click on the cart dropdown menu and confirm that the menu is displayed properly.
* Click on the login dropdown menu and confirm that it displays correctly.
* Navigate to the view more details page and confirm that all elements align properly.
* Confirm that texts resize appropriately

#### What are relevant pivotal tracker stories?
#115166123

#### Screenshots
![screen shot 2016-05-31 at 1 52 13 pm](https://cloud.githubusercontent.com/assets/7263503/15675025/d044667a-2737-11e6-84d5-e6df412bfb3d.png)



![screen shot 2016-05-31 at 1 53 08 pm](https://cloud.githubusercontent.com/assets/7263503/15675032/d69f4062-2737-11e6-8156-0138a4a88ea3.png)



![screen shot 2016-05-31 at 1 53 26 pm](https://cloud.githubusercontent.com/assets/7263503/15675038/dcc18360-2737-11e6-879f-e09538d9b509.png)



![screen shot 2016-05-31 at 1 54 10 pm](https://cloud.githubusercontent.com/assets/7263503/15675043/e36824b2-2737-11e6-9f99-0130bc62c1b2.png)
